lib/Console/Traits/ParallelizationBase.php

```diff
@@ -16,17 +16,25 @@
 
 namespace Pimcore\Console\Traits;
 
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Webmozarts\Console\Parallelization\Input\ParallelizationInput;
 
 if (trait_exists('\Webmozarts\Console\Parallelization\Parallelization')) {
     trait ParallelizationBase
     {
         use \Webmozarts\Console\Parallelization\Parallelization
         {
-            \Webmozarts\Console\Parallelization\Parallelization::configureParallelization as parentConfigureParallelization;
             \Webmozarts\Console\Parallelization\Parallelization::execute as parentExecute;
         }
+
+        protected function configureCommand(Command $command): void
```

Comment:
why not keep it static? as the called method from Webmozarts is also static.

Reply:
Should we also make it public then?

Reply:
`protected` is fine, since only inheriting commands will call this method.